ZBED4.py

```diff
@@ -0,0 +1,257 @@
+import Bio.SeqIO as SeqIO
+import Bio.SeqUtils as SeqUt
+
+class BiologicalSequence:
```

Comment:
По заданию предполагалось, что класс `BiologicalSequence` - абстрактный. Соответственно, наполнение методов должно быть реализовано в дочерних классах (в данном случае в `NucleicAcidSequence`).
Ну а этого покемона написать как-то так:


```suggestion
from abc import ABC, abstractmethod

class BiologicalSequence(ABC):
     @abstractmethod
     def len(self) -> int:
         pass
         
    @abstractmethod
    def check_alphabet(self) -> bool:
    """docstring"""
        pass
    
     ...
        

```